devops/openshift/api-deploy.yml

```diff
@@ -323,13 +326,13 @@ parameters:
   - name: PORT
     required: true
   - name: DB_SERVICE
-    value: patroni-master
+    value: pgbouncer-host
```

Comment:
I see. This is now an auto generated host name coming from secrets. 
![image](https://github.com/user-attachments/assets/71c5d507-459e-4e74-96d7-03d08e64930d)

Reply:
yes